src/test/java/com/revature/studyforce/flashcard/service/AnswerServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package com.revature.studyforce.flashcard.service;
+
+import com.revature.studyforce.flashcard.dto.AnswerDTO;
+import com.revature.studyforce.flashcard.model.Answer;
+import com.revature.studyforce.flashcard.model.Flashcard;
+import com.revature.studyforce.flashcard.repository.AnswerRepository;
+import com.revature.studyforce.flashcard.repository.FlashcardRepository;
+import com.revature.studyforce.user.model.Authority;
+import com.revature.studyforce.user.model.User;
+import com.revature.studyforce.user.repository.UserRepository;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Test class for AnswerService {@link AnswerService}
+ * @author Edson Rodriguez
+ */
+@SpringBootTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@TestPropertySource(locations = "classpath:test-application.properties")
+class AnswerServiceTest {
+
+    @MockBean
+    private UserRepository userRepository;
+
+    @MockBean
+    private FlashcardRepository flashcardRepo;
+
+    @MockBean
+    private AnswerRepository answerRepository;
+
+    @Autowired
+    private AnswerService answerService;
+
+    @Test
+    void givenAnswer_whenCreateAnswer_shouldReturnAnswerDTO(){
+        User user = new User(0,"edson@revature.com","password","Edson","Rodriguez",true,false,false, Authority.USER, Timestamp.valueOf(LocalDateTime.now()),Timestamp.valueOf(LocalDateTime.now()));
```

Comment:
Using outdated user model, please correct to user model with 1 name field and no password field